packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -384,14 +360,9 @@ export default function App() {
             {...rainbowKitProviderProps}
           >
             <ConnectedChainSyncer />
-            <WelcomeDialog />
-            <NetworkReady>
-              <AppContextProvider>
-                <Injector>
-                  <AppContent />
-                </Injector>
-              </AppContextProvider>
-            </NetworkReady>
+            <AppContextProvider>
+              <AppContent />
```

Comment:
combined the UI logic in `NetworkReady`, `Injector`, and `AppContent` into `AppContent`